src/consumer/consumer.js

```diff
@@ -603,6 +630,10 @@ export default class Consumer {
         }
       });
     }
+
+    // add workspaces if flag is true
+    if (this.bitJson.manageWorkspaces) await this.addWorkspacesToPackageJson();
```

Comment:
you should check that useWorkspaces is true and that packageManager is yarn. otherwise there is no reason to add workspaces

Reply:
we should pass the path in case the import is done using -p with a custom path, in this case, we will need to specifically add this path to the workspaces because it won't be under the default directories.